index.d.ts

```diff
@@ -284,6 +284,24 @@ declare namespace contextMenu {
 			dictionarySuggestions: MenuItemConstructorOptions[],
 			event: ElectronEvent
 		) => MenuItemConstructorOptions[];
+
+		/**
+		Called before context menu instance is created. Triggered by `menu-will-show` from `Menu` class.
+
+		Official documentation: https://www.electronjs.org/docs/latest/api/menu#event-menu-will-show
```

Comment:
I don't really see the point in mentioning `menu-will-show` or linking to its docs. The docs there are not very useful.

Reply:
Removed unnecessary mention & link to docs.